src/plugins/home/server/services/sample_data/usage/collector_fetch.ts

```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 
+import { SearchResponse } from 'elasticsearch';
```

Comment:
I think this should be imported from the new client `@elastic/elasticsearch` instead

Reply:
When I change the import location, my IDE errors with ```Module '"../../../../../../../node_modules/@elastic/elasticsearch"' has no exported member 'SearchResponse'.ts(2305)```